ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/winc/cucushift-installer-rehearse-aws-ipi-winc-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-winc
+  steps:
+    env:
+      SUB_CHANNEL: stable
+      SUB_INSTALL_NAMESPACE: openshift-windows-machine-config-operator
+      SUB_PACKAGE: windows-machine-config-operator
+      SUB_TARGET_NAMESPACES: '!install'
+      SUB_SOURCE: "qe-app-registry"
+      WINDOWS_NODE_REPLICAS: 2
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-winc-provision
+    test:
+    - ref: cucushift-installer-check
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-winc-deprovision
```

Comment:
Should use `cucushift-installer-rehearse-aws-ipi-deprovision`, If there is no special operation, we don't need to define additional chain

Reply:
@LiangquanLi930 So, that means I can get rid of the whole deprovision folder right?